ui/pages/Dashboard/Dashboard.jsx

```diff
@@ -1,11 +1,243 @@
 import React from 'react'
```

Comment:
this is just the moved/renamed ProjectsTable.jsx , nothing actually changed in this file